pkg/providers/tinkerbell/hardware/fmt.go

```diff
@@ -0,0 +1,11 @@
+package hardware
+
+import "fmt"
+
+func formatBMCRef(m Machine) string {
```

Comment:
Ignore these - they're used in a subsequent PR. They're dead because of some deletions.